### DISCUSSION TITLE
XmippTomo independient of xmipp?

### DISCUSSION THREAD

**Discussion by albertmena** [I2PC/scipion-em-xmipptomo#126] [Ideas]
We propouse this two posibilies:

- Add the installation of Xmipp in the installation of the plugin XmippTomo (because it needs Xmipp)
- Create new repository name XmippTomo with all the C++ code of xmippTomo and other named scipion-em-xmippTomo with the protocols, also change the installator and add the installation of xmippTomo and maybe xmippCore, not sure if also xmipp...

The idea, isolate xmippTomo to simplifie Xmipp to reduce the complexity and the time consumed installing it.

**Comment by cossorzano** [comment]
Either way is fine with me. What I would have as a directive is to avoid duplicating code. Duplication creates a lot of problems and prevents benefitting from debugging, code improving, acceleration, ...

**Comment by albertmena** [comment]
Maybe some dependencies with xmipp/.../data. Think about modular this data folder
Also depend on xmippCore bit is independient
